containers/scripts/generate_integration_test_coverage.sh

```diff
@@ -30,16 +30,21 @@ main() {
 
     ./containers/scripts/init-db.sh --full-dev --minimal "$FORCE"
 
+    # The container may have a different arch than the host!  Moreover, uname -m
+    # on macOS on Apple Silicon reports arm64 which is not recognised by Fedora.
+    ARCH=$(podman exec osh-client uname -m)
```

Comment:
Have you tested this command on Fedora? It was leaving a trailing carriage return on my system, which was causing `osh-cli` command to fail.

Reply:
I tried it on Fedora and it works. The trailing carriage return only appears when you run the command interactively.

Reply:
Oh, this is interesting:
```console
$ podman run centos:stream8 uname -m | hexdump -C
00000000  61 61 72 63 68 36 34 0a                           |aarch64.|
00000008

$ podman run -it centos:stream8 uname -m | hexdump -C
00000000  61 61 72 63 68 36 34 0d  0a                       |aarch64..|
00000009
```
Sice this script is not using `-it` the returned string does not end with `\r\n` so it should be fine.